homeassistant/components/konnected.py

```diff
@@ -75,6 +76,7 @@
 
 ENDPOINT_ROOT = '/api/konnected'
 UPDATE_ENDPOINT = (ENDPOINT_ROOT + r'/device/{device_id:[a-zA-Z0-9]+}')
+SIGNAL_SENSOR_UPDATE = 'konnected.sensor_updated'
```

Comment:
Have the signal key take a parameter in the form of the entity_id. Then you create unique signals per entity and the dispatcher will be more effective.
```py
SIGNAL_ENTITY_UPDATE = 'update_konnected_{}'
```

Reply:
Ok good idea. I'll address that in a minute.